src/Symfony/Component/DependencyInjection/Loader/schema/dic/services/services-1.0.xsd

```diff
@@ -258,6 +258,7 @@
     <xsd:attribute name="id" type="xsd:string" />
     <xsd:attribute name="key" type="xsd:string" />
     <xsd:attribute name="on-invalid" type="invalid_sequence" />
+    <xsd:attribute name="trim" type="boolean" />
```

Comment:
`boolean` is a type that support both `xsd:boolean` and DIC parameters as value. This `trim` should not support DI parameters as it is a value used by the loader directly.

Reply:
Thank you for letting me know this difference! Given your explanation, does this mean we should keep the `xsd:boolean` on `exclude-self` as well?

Reply:
depends whether DIC parameters work there or no.

Reply:
I tried and it doesn't work. I'll leave `xsd:boolean` for `exclude-self` as well :+1: